imod/mf6/ims.py

```diff
@@ -427,6 +427,32 @@ def __init__(
         self.dataset["no_ptc"] = no_ptc
         self._validate_init_schemata(validate)
 
+    def remove_model_from_solution(self, modelname: str) -> None:
+        models_in_solution = []
+        if "modelnames" in self.dataset.keys():
+            models_in_solution = self.dataset["modelnames"].values
+        if modelname not in models_in_solution:
+            raise ValueError(
+                f"attempted to remove model {modelname} from solution, but it was not found."
+            )
```

Comment:
This code is duplicate in ``add_model_to_solution``, I think you can move this to a method: ``get_models_in_solution``.

Reply:
done